Content.Server/Content.Server.csproj

```diff
@@ -67,6 +67,7 @@
     <Compile Include="AI\AimShootLifeProcessor.cs" />
     <Compile Include="EntryPoint.cs" />
     <Compile Include="GameObjects\Components\Doors\ServerDoorComponent.cs" />
+    <Compile Include="GameObjects\Components\Interactable\HandheldLightComponent.cs" />
```

Comment:
Should the new component go directly under "Interactable", or should a sibling directory to "Tools" be made to hold it?

Reply:
It's probably fine.